pages/pilot.vue

```diff
@@ -12,7 +12,11 @@
                 alerts.
```

Comment:
What are these changes to `pilot.vue`?  This file should not be changed in this PR I don't think.

Reply:
I did a `git pull` and it merged other commits in. Still remembering the right way to github. I can unstage any changes that aren't mine.